music21/noteworthy/translate.py

```diff
@@ -453,20 +486,39 @@ def translateNote(self, attributes):
         durationInfo = attributes['Dur']
         pitchInfo = attributes['Pos']
 
-        n = note.Note()   # note!
+        p = self.getPitchFromPositionInfo(pitchInfo)
+
+        if self.currentClef == 'PERCUSSION':
+            n = note.Unpitched()
+            n.fromDisplayPitch(p)
+        else:
+            n = note.Note()   # note!
+            n.pitch = p
 
         # durationInfo
         self.setDurationForObject(n, durationInfo)
 
         # pitchInfo
         self.setTieFromPitchInfo(n, pitchInfo)
-        n.pitch = self.getPitchFromPositionInfo(pitchInfo)
 
         # if Lyrics
         if self.lyrics and self.lyricPosition < len(self.lyrics):
             n.addLyric(self.lyrics[self.lyricPosition])
 
-        self.currentMeasure.append(n)
+        self.getCurrentVoice().append(n)
+
+    # current voice has shorter duration. If no voices, return current measure
+    def getCurrentVoice(self):
+        voice = None
+        shorter = 999
```

Comment:
this should be a float so that typing works for it.  And typed as `OffsetQL` since it can also be a Fraction.  `float('inf')` would be the best way of implying that any voice is shorter than this.

Reply:
and rename as `shortestQL`